### PR TITLE
Update Threat Grid for new API

### PR DESCRIPTION
**Pull request by saharm1**

### CODE REVIEW

File: Integrations/integration-Threat_Grid.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 commonfields:
-  id: Threat Grid
+  id: Threat Grid New
```
Comment (Itay4) [inline Integrations/integration-Threat_Grid.yml]: Why did you add `New` to `name` and `id` fields? As far as I understand this should keep backward compatibility.

**Comment by DeanArbel** [comment]
Updated documentation: https://github.com/demisto/etc/issues/13749

**Comment by DeanArbel** [comment]
documentation: https://github.com/demisto/etc/issues/13749

**Comment by DeanArbel** [comment]
relates to: https://github.com/demisto/etc/issues/8989
